packages/graphql/src/translate/create-where-and-params.ts

```diff
@@ -63,6 +64,37 @@ function createWhereAndParams({
             (x) => key.startsWith(x.fieldName) && x.typeMeta.name === "Duration"
         );
 
+        if (key.endsWith("Aggregate")) {
+            const [, reversedFieldName] = key
+                .split("")
+                .reverse()
+                .join("")
+                .split("Aggregate".split("").reverse().join(""));
+            const fieldName = reversedFieldName.split("").reverse().join("");
```

Comment:
Am I missing something here, or is this a very complicated way to drop the `Aggregate` at the end?

What about `key.replace("Aggregate", "")` or even `key.slice(0, -1*"Aggregate".length)` if we really want to make sure it's only the last bit that gets stripped?

Reply:
Yes, I don't know why I made this soo complicated lol :O.